tests/random.py

```diff
@@ -149,6 +149,18 @@ def random_Interferometer(draw, num_modes, trainable=False):
     return Interferometer(num_modes=num_modes, orthogonal_trainable=trainable)
 
 
+@st.composite
+def random_Interferometer_param(draw, trainable=False):
+    n_modes = [0,1,2]
+    modes = [2,5,6]
+    try:
+        Interferometer(num_modes=num_modes, orthogonal_trainable=trainable, modes=modes)
+    except:
+        pass
+    else:
+        raise AssertionError("No error here!")
```

Comment:
Thanks @sylviemonet! I think this should be placed in `tests/test_lab/test_gates_fock.py`. Also, pytest has a functionality to test error scenarios — you can use it as follows:
```suggestion
def random_Interferometer_param():
    n_modes = [0,1,2]
    modes = [2,5,6]
    with pytest.raises(ValueError):
        Interferometer(num_modes=num_modes, modes=modes)
```
here `pytest.raises` will succeed if `ValueError` is raised and fail otherwise.

Reply:
Thank you so much for this kind help. (I thought MM uses the Hypothesis for testing without Pytest...)